star-rx72n-firmware/lib/rx_bus/src/rx_bus_config.c

```
@@ -34,9 +34,8 @@ typedef uint8_t rx_pin_t;
  * @return k_rx_ok on success
  * @return k_rx_err_invalid_arg if port or pin_num is out of range
  */
-static rx_err_t internal_validate_port_pin(const rx_port_t port,
-                                           const rx_pin_t  pin_num,
-                                           const char*     context_tag)
+static rx_err_t
+internal_validate_port_pin(const rx_port_t port, const rx_pin_t pin_num, const char* context_tag)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_validate_port_pin' of similar type are easily swapped by mistake

(bugprone-easily-swappable-parameters)